tools/hsmtool.c

```diff
@@ -33,6 +35,7 @@ static void show_usage(const char *progname)
 	       "<path/to/hsm_secret> [hsm_secret password]\n");
 	printf("	- guesstoremote <P2WPKH address> <node id> <tries> "
 	       "<path/to/hsm_secret> [hsm_secret password]\n");
+	printf("	- generatehsm <path/to/hsm_secret>\n");
```

Comment:
```suggestion
	printf("	- generatehsm <path/to/new//hsm_secret>\n");
```
To imply it's not existing yet ?

Reply:
I am not used to that nomenclature but is it common for new files to be described as that?

Reply:
Absolutely not, entirely baked by myself (though i'm almost sure to have read that somewhere as well) because it's clearer in my mind. :)
Was more an opinion than a request!